{
  "messages":{
    "new":{
      "subject":"Neuer Teilnehmer",
      "text":"Ein neuer Teilnehmer hat sich am [date] [timeZone] mit Nummer [number] und Ipaddresse [Ip] angemeldet"
    },
    "invalidNumber":{
      "subject":"Ungültige Nummer",
      "text":"Ein Teilnehmer hat am [date] [timeZone] versucht sich mit der ungültigen Nummer [number] und IpAddresse [Ip] anzumelden"
    },
    "wrongDynIpPin":{
      "subject":"Falscher DynIp Pin",
      "text":"[Ip] hat am [date] [timeZone] zum [counter]ten mal mit einer falschen pin versucht die DynIp für [number]([name]) zu ändern"
    },
    "wrongDynIpType":{
      "subject":"Falscher DynIp Typ",
      "text":"[Ip] hat am [date] [timeZone] versucht die DynIp für [number]([name]) zu ändern, welcher keine DynIp unterstützt (Typ: [type])"
    },
    "ipV6DynIpUpdate":{
      "subject":"ipv6 DynIp Änderung",
      "text":"[Ip] hat am [date] [timeZone] versucht die DynIp für [number]([name]) mit einer ipv6 zu ändern"
    },
    "wrongServerPin":{
      "subject":"Falscher Serverpin",
      "text":"[Ip] hat am [date] [timeZone] versucht mit einer falschen Serverpin den Server zu syncronisieren"
    },
    "ServerError":{
      "subject":"Verbindung zum Server fehlgeschlagen",
      "text":"Der Server [host]:[port] konnte am [date] [timeZone] nicht erreicht werden. Fehlerzähler: [errorCounter] Fehlercode: [lastError]"
    },
    "ServerErrorOver":{
      "subject":"Störung Ende",
      "text":"Der Server: [host]:[port] konnte am [date] [timeZone] wieder erreicht werden. Fehlerzähler war: [errorCounter]"
    },
    "Nicht abgefangener Ausnahmezustand":{
      "subject":"Uncaught Exception",
      "html":"Fehler:\n[exception]"
    }
  }
}